x-pack/plugins/reporting/public/plugin.tsx

```diff
@@ -158,8 +158,8 @@ export class ReportingPublicPlugin implements Plugin<void, void> {
     const { http, notifications } = core;
     const apiClient = new ReportingAPIClient(http);
     const streamHandler = new StreamHandler(notifications, apiClient);
-    const { interval } = this.config.poll.jobsRefresh;
-
+    const { interval: intervalRaw } = this.config.poll.jobsRefresh;
+    const interval = typeof intervalRaw === 'number' ? intervalRaw : intervalRaw.asMilliseconds();
```

Comment:
I see this `typeof` check is in a few places, is there anyway we can consolidate this (maybe in our config module) to do this on initialization of the config object? That way we're not forcing downstream modules to do their own type checking